src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -131,6 +131,8 @@ public void statusUpdate(SchedulerDriver driver, TaskStatus status) {
           break;
         case JOURNAL_NODES :
           if (liveState.getJournalNodeSize() == conf.getJournalNodeCount()) {
+            // TODO move the reload to correctCurrentPhase and make it idempotent
+            reloadConfigsOnAllRunningTasks(driver);
```

Comment:
Question: Why do we also need to reloadConfigs after JNs, if we're already doing it for the START_NN case? In case a single JN relocates while the NNs stay up (so that correctCurrentPhase skips from Phase.JN to Phase.DN)?